### PR TITLE
[refs] ravel

### PR DESCRIPTION
**Pull request by kshitij12345**
As per title

### CODE REVIEW

File: torch/_refs/__init__.py [hunk 1]
```diff
@@ -1849,6 +1850,10 @@ def view(a: TensorLikeType, shape: ShapeType) -> TensorLikeType:
     return _reshape_view_helper(a, shape, allow_copy=False)
 
 
+def ravel(a: TensorLikeType) -> TensorLikeType:
+    return reshape(a, (-1,))
```
Comment (kshitij12345) [inline torch/_refs/__init__.py:1854]: IIRC, ref.reshape always returns a contiguous copy

File: torch/testing/_internal/common_methods_invocations.py [hunk 2]
```diff
@@ -16053,6 +16058,12 @@ def error_inputs_mean(op_info, device, **kwargs):
            supports_forward_ad=True,
            supports_fwgrad_bwgrad=True,
            sample_inputs_func=sample_inputs_ravel,
+           skips=(
+               # the stride of the tensor was modified directly without going through the PyTorch dispatcher.
+               DecorateInfo(unittest.expectedFailure, 'TestCompositeCompliance', 'test_backward'),
+               # the stride of the tensor was modified directly without going through the PyTorch dispatcher.
+               DecorateInfo(unittest.expectedFailure, 'TestCompositeCompliance', 'test_forward_ad'),
+           )
```
Comment (kshitij12345) [inline torch/testing/_internal/common_methods_invocations.py:16066]: The tests failed after adding a new non-contig input

File: torch/_refs/__init__.py (hunk 1 above)
Comment (mruberry) [reply torch/_refs/__init__.py:2182]: I don't think so -- it should work just like `torch.reshape` and sometimes return a view

File: torch/testing/_internal/common_methods_invocations.py (hunk 2 above)
Comment (mruberry) [reply torch/testing/_internal/common_methods_invocations.py:16215]: Because noncontiguous creation in `make_tensor` relies on `index_select`

Comment (mruberry) [reply torch/testing/_internal/common_methods_invocations.py:16215]: Right, sorry, those particular tests are failing for other reasons

Comment (kshitij12345) [reply torch/testing/_internal/common_methods_invocations.py:16215]: Sorry. I see that I commented at wrong line.

**Review by mruberry — APPROVED** [review]
Cool! Thanks @kshitij12345

**Pull request closed without merging**